x-pack/plugins/apm/server/lib/rum_client/get_client_metrics.ts

```diff
@@ -25,17 +24,14 @@ export async function getClientMetrics({
   const projection = getRumPageLoadTransactionsProjection({
     setup,
     urlQuery,
+    checkFetchStartFieldExists: false,
```

Comment:
So this should be only excluded for page views, for other aggs like frontend/backend time, we continue to want to use this.

Reply:
also we need to exclude this filter for page views chart.